backend/tests/test_equity_collateral_integration.py

```
@@ -0,0 +1,314 @@
+    return ("\n".join([header, *rows]) + "\n").encode("utf-8")
+
+
+# 100 shares bought directly — the imported-collateral case under test.
+STOCK_BUY_100_ROW = "03/01/2026,Buy,AAPL,APPLE INC,100,150.00,1.00,(15001.00)"
+# Sell all 100 imported shares back out (AC7b).
+STOCK_SELL_100_ROW = "03/10/2026,Sell,AAPL,APPLE INC,100,160.00,1.00,15999.00"
+# Sell 60 of the 100 imported shares, leaving 40 (optional partial-sell proof).
+STOCK_SELL_60_ROW = "03/10/2026,Sell,AAPL,APPLE INC,60,160.00,1.00,9599.00"
+
+
+@pytest.fixture()
```

Ruff (0.15.17)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)